.ci/update-github-io.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z $1 ]]; then
+  echo "version is not set"
+  echo "Usage: update-github-io.sh <version>"
+  exit 1
+fi
+
+TARGET_VERSION=$1
+echo TARGET_VERSION="$TARGET_VERSION"
+
+echo "Go to folder where site was build and sources are already at required tag"
+cd target/checkout
+
+echo "Generating web site"
+mvn -e --no-transfer-progress site -Pno-validations -Dmaven.javadoc.skip=false
+
+echo "Come back repo folder"
+cd ../../
+
+echo "Switch to checkstyle.github.io repository"
+if [ -d "../checkstyle.github.io" ] ; then
+  cd ../checkstyle.github.io
+else
+  cd ../
+  echo "Clone by ssh only to avoid passwords on push ..."
+  git clone git@github.com:checkstyle/checkstyle.github.io.git
+  cd checkstyle.github.io
+fi
+git rm -rf *
+git checkout HEAD -- CNAME
+cp -R ../checkstyle/target/checkout/target/site/* .
```

Comment:
No checkout folder repo already checkouted to right code.

Reply:
Changes Done